includes/api/save-abandonment-data.php

```diff
@@ -59,6 +60,8 @@ function saveCartAbandonmentData(WP_REST_Request $request)
     $result['response']    = "";
     $result['status_code'] = 400;
 
+    abandonedPluginHook($razorpayData); // do_action to notify/send the abandonedCart data to 3rd party plugins
```

Comment:
This function should call right after the try-catch block if there is no other dependency.